### PR TITLE
Update SwiSBox XML

### PR DESCRIPTION
**Pull request by d-spoerri-swistec**
As discussed via E-Mail with @ergo-furrer 

Changes:
- Change device category to SubMeterElectricity
- Fix register numbers
- Fix functional profile names, and texts
- Add apparent energy and reactive power
- Remove duplicate entry for apparent power

I've tested this XML (with the relay FPs removed) using the java library. Please take a close look at it, as I'm not aware of the specific requirements regarding these XMLs.

### CODE REVIEW

File: XMLInstances/ExtInterfaces/SGr_02_mmmm_dddd_SwiSBox_Actuator.xml [hunk 1]
```diff
@@ -1390,6 +1402,7 @@
           </dataPointList>
         </functionalProfileListElement>
       </functionalProfileList>
+      </functionalProfileList>
```
Comment (ArndGerns) [inline XMLInstances/ExtInterfaces/SGr_02_mmmm_dddd_SwiSBox_Actuator.xml:1405]: This is a syntax error (double `</functionalProfileList>`).

File: XMLInstances/ExtInterfaces/SGr_02_mmmm_dddd_SwiSBox_Actuator.xml [hunk 2]
```diff
@@ -426,253 +424,259 @@
...
-                <unit>KILOWATTS</unit>
+                <unit>KILOVOLT_AMPERES_REACTIVE</unit>
                 <legibleDescription>
-                  <textElement>Erfassung der gesamten Wirkleistung</textElement>
+                  <textElement>Erfassung der gesamten Blindleistung</textElement>
                   <language>de</language>
                 </legibleDescription>
+                <legibleDescription>
+                  <textElement>Total reactive power measurement</textElement>
+                  <language>fr</language>
```
Comment (ArndGerns) [inline XMLInstances/ExtInterfaces/SGr_02_mmmm_dddd_SwiSBox_Actuator.xml:467]: Why French?

**Comment by ArndGerns** [comment]
At some places the text formatting was removed - in the following file, I have restored the formatting (renamed to `.txt` as it was not possible to add a `.xml`).

[SGr_02_mmmm_dddd_SwiSBox_SubMeterElectricity.xml](https://github.com/SmartGridready/SGrSpecifications/files/13759827/SGr_02_mmmm_dddd_SwiSBox_SubMeterElectricity.txt)

Furthermore, I have removed the relais for now. AFAIK this was discussed with Maike Schubert / Deklarationsstelle.

If the changed version is OK for you, we can check it in into the main repository. Then you need to discuss with the Deklarationsstelle to get it published.

**Comment by d-spoerri-swistec** [comment]
Thanks for your feedback and corrections. It is correct, that the relays will not be part of the initial declaration, but I've re-added them as I was not sure if we should leave them in the XML (I must have also added the duplicate tag while doing that).
I've added all of your changes to my XML as well now, so from my view, a merge should be ok now.

**Review by ArndGerns — COMMENTED** [review]
Hi @d-spoerri-swistec,

the content of the XML is good now. There are two things, that should be checked:

* File name should be `SGr_02_mmmm_dddd_SwiSBox_SubMeterElectricity.xml` as the category should be part of the file name
* The file has unix-style line endings with only NL - other files have windows-style line endings with CR+NL. Need to see whether this gives problems.

If you're OK, we can merge it, and I will do these two adaptions.

Best regards,
Arnd.

**Comment by d-spoerri-swistec** [comment]
@ArndGerns thanks for your feedback and all the support, II don't think there's any more changes from our side, so this may be merged. Thanks!

**Review by ArndGerns — APPROVED** [review]

**Comment by LTA-WKP** [comment]
@ArndGerns: As discussed the IDs for the file names
UnternehmensID: 4893879785
DeklarationsID: 8288144069

BR
Luca

**Comment by ArndGerns** [comment]
Hi @LTA-WKP, @wkp-maike,

I have renamed the EID XML accordingly, It should now be ready for declaration.

Best regards,
Arnd.

**Pull request merged**